environments/dev/terraform/main.tf

```diff
@@ -14,12 +14,19 @@ provider google-beta {
   alias = "clinvar"
 }
 
+provider google-beta {
+  project = "broad-dsp-monster-encode-dev"
+  region = "us-central1"
```

Comment:
Need to re-review the TF... ENCODE's systems are all setup in a northwestern US region of AWS, it could be good to co-locate some or all of our GCP infrastructure.